app/forms/degree_form.rb

```diff
@@ -87,7 +88,7 @@ def stash
   def save!
     return false unless valid?
 
-    degree.attributes = fields
+    degree.attributes = fields.merge(uuids_fields)
```

Comment:
nit: `uuids_fields` feels a bit clunky. Maybe just `uuids` reads a bit better?
```suggestion
    degree.attributes = fields.merge(uuids)
```